tests/integration-tests/configs/common/common.yaml

```diff
@@ -806,7 +806,7 @@ update:
     dimensions:
       - regions: [ "eu-west-2" ]
         schedulers: [ "slurm" ]
-        oss: ["alinux2"]
+        oss: ["alinux2", "rhel8"]
```

Comment:
let's replace alinux2 with rhel8 for all of them, since the update code is not OS specific, to avoid increasing the integ test time